src/dmd/hdrgen.d

```diff
@@ -2044,6 +2044,14 @@ public:
         buf.writeByte(')');
     }
 
+    override void visit(CompoundLiteralExp e)
+    {
+        buf.writeByte('(');
+        typeToBuffer(e.type, null, buf, hgs);
+        buf.writeByte(')');
+        e.initializer.initializerToBuffer(buf, hgs);
```

Comment:
This code is not covered by tests.

Reply:
Right, because the only way to test them is with error messages.